boa/src/builtins/error/reference.rs

```diff
@@ -10,24 +10,50 @@
 //! [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ReferenceError
 
 use crate::{
-    builtins::{function::make_builtin_fn, function::make_constructor_fn},
-    object::ObjectData,
+    builtins::BuiltIn,
+    object::{ConstructorBuilder, ObjectData},
     profiler::BoaProfiler,
+    property::Attribute,
     Context, Result, Value,
 };
 
 #[derive(Debug, Clone, Copy)]
 pub(crate) struct ReferenceError;
 
-impl ReferenceError {
-    /// The name of the object.
-    pub(crate) const NAME: &'static str = "ReferenceError";
+impl BuiltIn for ReferenceError {
+    const NAME: &'static str = "ReferenceError";
+
+    fn attribute() -> Attribute {
+        Attribute::WRITABLE | Attribute::NON_ENUMERABLE | Attribute::CONFIGURABLE
+    }
+
+    fn init(context: &mut Context) -> (&'static str, Value, Attribute) {
+        let _timer = BoaProfiler::global().start_event(Self::NAME, "init");
 
+        let error_prototype = context.standard_objects().error_object().prototype();
+        let attribute = Attribute::WRITABLE | Attribute::NON_ENUMERABLE | Attribute::CONFIGURABLE;
+        let range_error_object = ConstructorBuilder::with_standard_object(
```

Comment:
`range_error_object` should be `reference_error_object`

Reply:
and here